CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 Unreleased section should follow [Release Toolkit](https://github.com/newrelic/release-toolkit#render-markdown-and-update-markdown).
 
+## Held
```

Comment:
not sure how long will be held, but in case is too long, perhaps we could revert the commit to avoid blocking sec updates

Reply:
It is already identified, we'll fix it today